scripts/localize_podfile.swift

```diff
@@ -45,11 +55,12 @@ while url.path != "/", url.lastPathComponent != "firebase-ios-sdk" {
 
 let repo = url
 let lines = fileContents.components(separatedBy: .newlines)
-var outBuffer = "source 'https://github.com/firebase/SpecsStaging.git'\n" +
-  "source 'https://cdn.cocoapods.org/'\n"
+var outBuffer =
+  "source 'https://github.com/firebase/SpecsStaging.git'\n"
+    + "source 'https://cdn.cocoapods.org/'\n"
 for line in lines {
-  var newLine = line
-  let tokens = line.components(separatedBy: [" ", ","] as CharacterSet)
+  var newLine = line.trimmingCharacters(in: .whitespacesAndNewlines)
+  let tokens = newLine.components(separatedBy: [" ", ","] as CharacterSet)
   if tokens.first == "pod" {
```

Comment:
I think this can be `if tokens.first == "pod" && !releaseTesting {` and then undo the changes inside the test.

The only change necessary for testing the staged pods should be adding source `'https://github.com/firebase/SpecsStaging.git'` to the Podfile.